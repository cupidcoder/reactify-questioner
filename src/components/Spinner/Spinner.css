.Spinner {
  position: absolute;
  width: 1.5rem;
  top: 1rem;
  height: 1.5rem;
  right: 0;
  padding-right: 0.3rem;
}
