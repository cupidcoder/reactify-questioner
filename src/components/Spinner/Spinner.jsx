import React from 'react';
import spinner from '../../assets/gifs';
import style from './Spinner.css';

const Spinner = () => (
  <img className={style.Spinner} src={spinner} alt='spinner' />
);

export default Spinner;
