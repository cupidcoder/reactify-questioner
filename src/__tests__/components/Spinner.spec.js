import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../components/Spinner/Spinner';

describe('Spinner', () => {
  it('Should correctly save snapshot of the Spinner', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
