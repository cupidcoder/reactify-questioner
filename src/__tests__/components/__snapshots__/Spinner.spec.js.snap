// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Should correctly save snapshot of the Spinner 1`] = `ShallowWrapper {}`;
